src/NHSD.BuyingCatalogue.Solutions.Persistence/Repositories/SolutionRepository.cs

```diff
@@ -29,7 +29,7 @@ public sealed class SolutionRepository : ISolutionRepository
                             s.ClientApplication,
                             s.Hosting,
                             s.LastUpdated AS SolutionDetailLastUpdated,
-                            f.IsFoundation
+                            (SELECT MAX(CAST(IsFoundation AS int)) FROM dbo.FrameworkSolutions WHERE SolutionId = @id GROUP BY SolutionId) AS IsFoundation
```

Comment:
We should quality `IsFoundation` with the table alias so it becomes `f.IsFoundation`.